<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gs_AcquisitionDialog</class>
 <widget class="QDialog" name="gs_AcquisitionDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>495</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>463</width>
    <height>900</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
    <pointsize>14</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>采集卡设置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>设备:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_comebox_Device">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>开始通道:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_comebox_StartChannel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>通道个数:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_comebox_ChannelCount">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>采样频率:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineED_ClockRate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>单通道采样数:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_lineED_SectionLength">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>431</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_AcquisitionSure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_AcquisitionDefault">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>380</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>默认值</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>m_comebox_Device</zorder>
  <zorder>label_3</zorder>
  <zorder>m_comebox_StartChannel</zorder>
  <zorder>label_4</zorder>
  <zorder>m_comebox_ChannelCount</zorder>
  <zorder>label_5</zorder>
  <zorder>m_lineED_ClockRate</zorder>
  <zorder>label_6</zorder>
  <zorder>m_lineED_SectionLength</zorder>
  <zorder>m_btn_AcquisitionSure</zorder>
  <zorder>m_btn_AcquisitionDefault</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
